base/scopedvalues.jl

```diff
@@ -54,7 +55,22 @@ Base.eltype(::ScopedValue{T}) where {T} = T
 """
     isassigned(val::ScopedValue)
 
-Test whether a ScopedValue has an assigned value.
+Test whether a `ScopedValue` has an assigned value.
+
+See also: [`ScopedValues.with`](@ref), [`ScopedValues.@with`](@ref), [`ScopedValues.get`](@ref).
```

Comment:
Due to #53004  you might need `Base.ScopedValues`

Reply:
The Iterators module uses references like ```[`Iterators.drop`](@ref)```, so I think this might be ok the way it is.